veps/sig-network/passt-migration-proposal.md

```diff
@@ -0,0 +1,185 @@
+2. Before migration finalizes (target node) in [finalizeMigration](https://github.com/kubevirt/kubevirt/blob/release-1.5/pkg/virt-handler/vm.go#L2714) to set and then clear the TCP_REPAIR socket option.
+
+##### Failure Scenarios:
+passt-repair can fail or timeout if:
+- The Unix Domain Socket created by passt isn't found.
+- Communication with passt is delayed or unresponsive.
+
+Either scenario likely results in broken TCP connections for the guest VM, requiring reconnection.
+- virt-handler does not fail migration on passt-repair errors.
+- passt currently does not abort migration upon TCP_REPAIR failure.
+
+**Note**: Success of passt-repair doesn't guarantee seamless migration alone; sticky IPs from the network provider and CNI are essential.
+
+#### Add passt-repair binary to virt-handler image
+The `passt-repair` binary must be added to the virt-handler container image. The simplest solution is to install the passt RPM. A more secure and storage-efficient solution is extracting only the passt-repair binary from the RPM and installing it standalone.
+
+### Implement vhost-user support in passt sidecar
+vhost-user enables high-performance communication between virtual machines and user-space processes, typically offloading VirtIO emulation to dedicated user-space processes (e.g., DPDK or passt). It improves I/O performance through zero-copy mechanisms and reduced context-switching overhead. Additionally, it is required for seamless TCP migration.
+
+vhost-user must be enabled in the DomainXML. Small code changes are required in passt sidecar code. The format is described [here](https://libvirt.org/formatdomain.html#vhost-user-connection-with-passt-backend).
+
+### Passt CNI
+The passt-cni process is called by `Multus` during primary network interface setup/removal. It runs two sysctl commands:
+1. Allow binding to all ports starting from 0.
+2. Set ping group range to virt-launcher's user ID.
+
+The [passt-binding directory](https://github.com/kubevirt/kubevirt/tree/release-1.5/cmd/cniplugins/passt-binding) will include:
+- A `Dockerfile` to build and publish the container image.
+- YAML manifests for a DaemonSet to deploy the CNI onto nodes.
+
+These artifacts already exist under the IPAM Controller repository and can be moved as-is from their [current location](https://github.com/kubevirt/ipam-extensions/tree/v0.1.10-alpha/passt).
+
+A manifest for applying a `network-attachment-definition` (NAD) CR named `netbindingpasst` pointing to the CNI will also be included:
+
+```yaml
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: netbindingpasst
+  namespace: {{ .namespace }}
+spec:
+  config: '{ "cniVersion": "0.3.1", "name": "netbindingpasst", "plugins": [{"type": "kubevirt-passt-binding"}]}'
+```
+
+**Note**: No changes are expected in the passt sidecar container.
+
+## API Examples
+No API changes are expected in kubevirt/kubevirt.
```

Comment:
Nevertheless, I think you should link to a document or provide the example how one will use it in practice.